platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBean.java

```diff
@@ -181,8 +183,13 @@ public void destroy() throws ApplicationServiceException {
   @Override
   public void installFeature(String feature) {
```

Comment:
✏️ I think we should privilege the `uninstallFeature` method as well. I'm going to link to a gist with my suggested version of this class on this review.